### ISSUE TITLE
Generic class or type alias that uses a TypeVarTuple followed by a TypeVar with a default value should be rejected

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#7443]
PEP 696 indicates that the following should be rejected, but pyright doesn't generate an error in this case.

```python
Ts = TypeVarTuple("Ts")
T5 = TypeVar("T5", default=bool)

class Foo5(Generic[*Ts, T5]): ...  # Type Error
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.354, which I just published.
